doc/src/user_guide.rst

```diff
@@ -1289,6 +1346,7 @@ Example::
 
     avg-p = p
     avg-p2 = p*p
+    fun-avg-varp = p2 - p*p
```

Comment:
So, just checking, the p here is the averaged p from avg-p, not the instantaneous p?

Reply:
Yes, instantaneous values here would not make sense and prefixing with avg- is confusing.

Reply:
Agreed.